packages/plugin-ext/src/main/node/plugin-deployer-impl.ts

```diff
@@ -296,40 +296,38 @@ export class PluginDeployerImpl implements PluginDeployer {
      * If there are some single files, try to see if we can work on these files (like unpacking it, etc)
      */
     public async applyFileHandlers(pluginDeployerEntries: PluginDeployerEntry[]): Promise<any> {
```

Comment:
Minor: A bit unrelated to the changes at hand, but shouldn't we just return `Promise<void>` here? Same for the `applyDirectoryFileHandlers` method below.

Reply:
https://github.com/eclipse-theia/theia/pull/12798/commits/e78a76148e43b02473c899f8090b2514365f2e22